tools/cooja/apps/avrora/build.gradle

```diff
@@ -0,0 +1,24 @@
+apply plugin: "java"
+
+jar.baseName = 'cooja_avrora'
+
+repositories {
+  jcenter()
+}
+
+sourceSets {
+  main {
+    java {
+      srcDir 'src'
+    }
+  }
+
+}
+
+
+dependencies {
+    compile project(":")
+    compile files("lib/avrora-small.jar")
```

Comment:
I somewhat (strongly) disagree. We have a circular dependency. The plugins need Cooja to compile and we don't want Cooja without the plugins[^1]. And, as they are in the same build-tree, I think that, while having those artifacts in a maven-repo is the intended way to go, there are no real advantages in this case.
Especially as Cooja and its plugins never had, and very likely will never have, reasonable versioning. Without you'll come into the Maven-Versioning-Hell, because all builds and checkouts will share a single jar in the maven-repository and you'll spend hours to find out why things a not working as intended. This happened to me on another project where I was working on multiple checkouts at the same time.
To wrap it up: I'm very in favor of (local) maven-repositories, but I don't think they fit Cooja's development-model.

^1: Although I implemented plugin-autodiscovery a while ago, last I checked their paths were still hard-coded in the config.